src/api/v1/models.py

```diff
@@ -136,39 +136,49 @@ class TagsBodyParams(BaseModel):
 class ResampleQueryParams:
     def __init__(
         self,
-        sample_rate: str = Query(..., description="Sample Rate", example=5),
-        sample_unit: str = Query(..., description="Sample Unit", examples={"second": {"value": "second"}, "minute": {"value": "minute"}, "hour": {"value": "hour"}, "day": {"value": "day"}}),
-        agg_method: str = Query(..., description="Aggregation Method", examples={"first": {"value": "first"}, "last": {"value": "last"}, "avg": {"value": "avg"}, "min": {"value": "min"}, "max": {"value": "max"}}),   
+        sample_rate: str = Query(..., description="Sample Rate as a numeric input", example=5, deprecated=True),
+        sample_unit: str = Query(..., description="Sample Unit can be one of the options: [second, minute, day, hour]", examples={"second": {"value": "second"}, "minute": {"value": "minute"}, "hour": {"value": "hour"}, "day": {"value": "day"}}, example=5, deprecated=True),
```

Comment:
Can you add a message to use the time interval fields instead in the description?